workflow/applications/gfs_forecast_only.py

```diff
@@ -27,9 +27,8 @@ def _get_app_configs(self, run):
         """
         Returns the config_files that are involved in the forecast-only app
         """
-
+        configs = ['fetch', 'stage_ic', 'fcst', 'arch', 'cleanup']
         options = self.run_options[run]
```

Comment:
`config.fetch` should only be added if `fetch_hpss` or `fetch_local` are `True`.

```suggestion
        configs = []
        options = self.run_options[run]
        if options.fetch_hpss or options.fetch_local:
            configs += ['fetch']

        configs += ['stage_ic', 'fcst', 'arch', 'cleanup']
```

Reply:
Made those modifications